mozci/push.py

```diff
@@ -1204,14 +1207,17 @@ def classify_regressions(
         groups_non_consistent_failure = {
             g.name
             for g in all_groups
-            if g.status != Status.PASS
-            and g.is_consistent_failure(
-                cross_config_counts[0] if cross_config_counts is not None else None,
-                consistent_failures_counts[0]
-                if consistent_failures_counts is not None
-                else None,
+            if groups_confirmed[g.name] is None
```

Comment:
I would say both if `is None` and `is False` here, you could do 
```suggestion
            if not groups_confirmed[g.name]
```

Reply:
end result is similar to this, but not exactly.